configure.ac

```diff
@@ -60,7 +60,7 @@ XORG_TESTSET_CFLAG(CWARNFLAGS, [-Wno-int-conversion])
 
 has_znow=yes
 
-case $host_os in
+case $target_os in
```

Comment:
This is wrong, `$host` is the correct value.
`$build` represents the machine the compiler will run on, `$host` the machine the compiled binaries will run on, and `$target` is the machine the run binary *will produce code for*. See the [Autoconf manual](http://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf.html#Canonicalizing) for reference.
`$target` is only needed for compilers and toolchain utilities.